priv/riak_kv.schema

```diff
@@ -967,3 +967,45 @@
   fun(Value) when is_integer(Value) andalso Value > 0 -> true;
      (_) -> false
   end}.
+
+%% @doc Register allowable origins (for CORS), specified as a list of ip
```

Comment:
Perhaps a question for @angrycub: should we make this `riak_kv.cors.allowable_origin` instead of `riak_kv.allowable_origin` to make its purpose explicit?